hub/auto/unstructured/image_classification.py

```diff
@@ -140,18 +151,24 @@ def structure(  # type: ignore
                 # if appending fails because of a shape mismatch, expand dims (might also fail)
                 try:
                     ds[images_tensor_map[set_name]].append(image)
+
                 except TensorInvalidSampleShapeError:
                     im = image.array
                     reshaped_image = np.expand_dims(im, -1)
                     ds[images_tensor_map[set_name]].append(reshaped_image)
 
-                except Exception as e:
-
-                    warnings.warn(
-                        f"[Skipping] Could not upload sample '{file_path}'. Reason: {e}"
+                except Exception:
+                    skipped_files.append(file_path.name)
+                    iterator.set_description(
+                        'Ingesting "%s" (%i files skipped)'
+                        % (self.source.name, len(skipped_files))
                     )
+                    ingested_file_count -= 2
```

Comment:
why is this -2 here? if we're skipping it and continuing the part of the loop that increments the count shouldn't this line not exist?

Reply:
thanks for that! I forgot to remove it after some testing 😅